api/task_lifecycle_client.go

```diff
@@ -59,31 +58,6 @@ func (c *TaskLifecycleClient) Scheme() string {
 	return c.url.Scheme
 }
 
-// CreateTaskSimple takes a task request and run option and sends this information to the client. It then returns
-// a task response object and any errors to the caller.
-// TODO: remove this to conform to interface
-func (c *TaskLifecycleClient) CreateTask(ctx context.Context, runOption string, req TaskRequest) (*oapigen.CreateTaskResponse, error) {
-	var run oapigen.CreateTaskParamsRun
-	switch runOption {
```

Comment:
Swagger validation happens in the generated client code, and that validates the run parameter is in the enum of supported options, no need for our additional validation logic.